### DISCUSSION TITLE
maa调用显卡的程度是什么样的呀？外接显卡坞的mac长时间运行依然会卡

### DISCUSSION THREAD

**Discussion by Bingreal** [MaaAssistantArknights/MaaAssistantArknights#7334] [Q&A]
事情是这样的，我的mbp外接了显示器和显卡坞，平常也能正常调用外接显卡玩一些大型点的游戏，但是使用maa+模拟器的时候，经常出现运行一段时间macbook的风扇开始狂转，但是显卡坞里的显卡古井无波，这样运行个十多分钟就开始感到模拟器明显变卡，有时候卡放不下去/技能开不出来。

想请问这种问题有没有解决办法呀？

**Comment by ABA2396** [comment]
maa没用到显卡。

**Comment by AjinKimxu** [comment]
你的外接显卡是a卡吧，a卡在模拟器上显存释放有问题长时间运行会卡（至少我的a卡是这样），换成n卡或者把模拟器跑在核显上就解决问题了
